BrowserKit/Sources/Common/Constants/StandardImageIdentifiers.swift

```diff
@@ -110,6 +110,18 @@ public struct StandardImageIdentifiers {
         public static let warning = "warningLarge"
         public static let warningFill = "warningFillLarge"
         public static let whatsNew = "whatsNewLarge"
+
+        public struct TrackingProtection {
```

Comment:
No sub-structs apart from the existing ones that are based on size should be added to `StandardImageIdentifiers`.